tests/bootstrap.py

```diff
@@ -26,7 +26,7 @@
 import sys
 
 pip_packages = [
-    "autest==1.7.3",
+    "autest==1.7.4",
```

Comment:
Any reason not to go to 1.8.0? One of the fixes in 1.8.0 fixes Process cleanup function dispatch. Without that, we don't recycle ports as expected from this cleanup function from autest-site/ports.py:

https://github.com/ezelkow1/trafficserver/blob/81x-autest/tests/gold_tests/autest-site/ports.py#L118

That bug eventually results in us running out of safe ports and we have to fall back to a less optimal "bind, get the port, release" mechanism which has a rare race condition which could lead to an EADDRINUSE error.

Reply:
sure, if it works, I was just trying to get a minimal viable set of changes to have this work properly.  I also didnt know if that would need wide sweeping changes to tests as well to go further since this is the 8.1.x branch, so was trying to avoid having to backport large sections of test changes

Reply:
Understood. It should just work. When we updated to 1.8.0 on master we didn't have to change the tests at all to accommodate the new version.